lib/components/admin/field-trip-list.js

```diff
@@ -174,11 +234,11 @@ class FieldTripRequestRecord extends Component {
               {this._getStatusIcon(outboundTripStatus)} Outbound
             </span>
           </span>
-          <span style={{display: 'block'}}>
+          <span style={{display: 'block', fontSize: '.9em'}}>
```

Comment:
Converting a lot of these components into styled components instead of having inline css might also improve performance. However, having a virtualized list for all these items would probably be even better.